DB/projects.json

```
@@ -3634,7 +3634,23 @@
           "API",
           "Vercel"
         ]
+      },
+      {
+        "link": "https://github.com/Harshu467/NFT",
+        "title": "NFT Market",
+        "description": "Master React Native by building a modern NFT Marketplace iOS and Android #ReactNative Application ",
+        "tech": [
+          "html",
+          "css",
+          "javascript",
+          "reactjs JS",
+          "useContext",
+          "API",
+          "Vercel",
+          "React-Native"
```

convert to lowercase

```suggestion
          "reactnative"
```